backend/src/main/java/com/webappsbusters/parcelmanagement/controller/ParcelController.java

```diff
@@ -16,18 +17,20 @@
 
     private final ParcelService parcelService;
     private final ParcelMapper parcelMapper;
```

Comment:
I think we should remove this old mapper, as it is not needed now. Remove ParcelMapper.class and this field